File: api/core/src/main/kotlin/data/UciteljData.kt
package data

import domain.Naslov
import domain.Oseba
import kotlinx.serialization.Serializable

@Serializable
data class UciteljData(
    val oseba: Oseba,
    val naslov_refs: List<Naslov> = listOf(),
    val kontakt_refs: List<KontaktData> = listOf()
)

File: app/src/app/core/services/api/models/ucitelj-data.ts
import { AuditData } from './audit-data';
import { KontaktData } from './kontakt-data';
import { NaslovData } from './naslov-data';
import { Oseba } from './oseba';
import { TestData } from './test-data';
import { UcenjeData } from './ucenje-data';
export interface UciteljData {
	audit_refs: Array<AuditData>
	kontakt_refs: Array<KontaktData>
	naslov_refs: Array<NaslovData>
	oseba: Oseba
	test_admin_refs: Array<TestData>
	ucenje_ucitelj_refs: Array<UcenjeData>
}